### PR TITLE
Fix HA documents

### PR DESCRIPTION
**Pull request by cfmcgrady**
### _Why are the changes needed?_

1. the default value of `kyuubi.ha.zookeeper.namespace` is `kyuubi`
2. the beeline's variable `zooKeeperNamespace` is case-sensitive
3. how to configure kyuubi ha

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cfmcgrady** [comment]
cc @yaooqinn

**Comment by yaooqinn** [comment]
`kyuubi.ha.enabled` is not necessary to me. As we can always support client connection via zookeeper

1. if the ha.zookeeper.quorum is empty, clients can use both serviceUri(hostname:10009) and zkAddress (hostname:2181) to access Kyuubi server
2. if the ha.zookeeper.quorum is specified to external zk, client scan still use both serviceUri(hostname:10009) and zkAddress (quorm-peer1:2181, qp2:2191) to access Kyuubi server(s)

**Comment by yaooqinn** [comment]
The rest of this PR for doc change looks good to me

**Comment by cfmcgrady** [comment]
> `kyuubi.ha.enabled` is not necessary to me. As we can always support client connection via zookeeper
> 
> 1. if the ha.zookeeper.quorum is empty, clients can use both serviceUri(hostname:10009) and zkAddress (hostname:2181) to access Kyuubi server
> 2. if the ha.zookeeper.quorum is specified to external zk, client scan still use both serviceUri(hostname:10009) and zkAddress (quorm-peer1:2181, qp2:2191) to access Kyuubi server(s)

SGTM, reverted it.

**Review by yaooqinn — APPROVED** [review]

File: docs/deployment/high_availability_guide.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Load balancing aims to optimize all Kyuubi service units usage, maximize through
 
 ![](../imgs/ha.png)
 
-With Hive JDBC Driver, a client can specify service discovery mode in JDBC connection string, i.e. `serviceDiscoveryMode=zooKeeper;` and set `zooKeeperNameSpace=kyuubiserver;`, then it can randomly pick one of the Kyuubi service uris from the specified ZooKeeper address in the `/kyuubiserver` path.
+With Hive JDBC Driver, a client can specify service discovery mode in JDBC connection string, i.e. `serviceDiscoveryMode=zooKeeper;` and set `zooKeeperNamespace=kyuubi;`, then it can randomly pick one of the Kyuubi service uris from the specified ZooKeeper address in the `/kyuubi` path.
 
-When we set `kyuubi.ha.enabled` to `true`, load balance mode is activated by default. Please make sure that you specify the correct ZooKeeper address via `kyuubi.ha.zookeeper.quorum` and `kyuubi.ha.zookeeper.client.port`.
+When we specify the ZooKeeper address via `kyuubi.ha.zookeeper.quorum`, load balance mode is activated by default. Please make sure that you specify the correct ZooKeeper address via `kyuubi.ha.zookeeper.quorum` and `kyuubi.ha.zookeeper.client.port`.
```
Comment (yaooqinn) [inline docs/deployment/high_availability_guide.md:34]: Can we also update L38

Comment (cfmcgrady) [reply docs/deployment/high_availability_guide.md]: the configurations listed here are out of date, removed, and add a link to https://kyuubi.apache.org/docs/latest/deployment/settings.html#ha.

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**